lib/Service/CleanupService.php

```diff
@@ -47,16 +48,20 @@ class CleanupService {
 	/** @var TagMapper */
 	private $tagMapper;
 
-	public function __construct(AliasMapper $aliasMapper,
-		MailboxMapper $mailboxMapper,
-		MessageMapper $messageMapper,
+	private MessageRetentionMapper $messageRetentionMapper;
+
+	public function __construct(AliasMapper            $aliasMapper,
+		MailboxMapper          $mailboxMapper,
+		MessageMapper          $messageMapper,
 		CollectedAddressMapper $collectedAddressMapper,
-		TagMapper $tagMapper) {
+		TagMapper              $tagMapper,
+		MessageRetentionMapper $messageExtraMapper) {
```

Comment:
```suggestion
		MessageRetentionMapper $messageRetentionMapper) {
```

Reply:
Done. I also fixed the weird alignment applied by PhpStorm.